cfme/tests/infrastructure/test_providers.py

```diff
@@ -492,3 +495,38 @@ def test_infra_discovery_screen(appliance):
                    "to_ip4": ips['to']})
         view.start.click()
         view.flash.assert_message(ips['msg'])
+
+
+@pytest.fixture
+def setup_provider_min_templates(request, appliance, provider, min_templates):
+    templates_yaml = len(provider.data.get('templates', {}))
+    if templates_yaml < min_templates:
+        pytest.skip(f'Number of templates defined in yaml for {provider} does not meet minimum '
+                    f'for test parameter {min_templates}, skipping and not setting up provider')
+    if len(provider.mgmt.list_host()) < min_templates:
```

Comment:
This should be `provider.mgmt.list_template()`

Reply:
Whoops, good catch. Updated as suggested. At least this was just part of reducing setup and not part of the actual test.

Reply:
list_template is not working. I am looking to see what format this is expecting.

Reply:
list_templates works.